apps/examples/tls_client/tls_client_main.c

```diff
@@ -831,6 +841,11 @@ int tls_client_cb(void *args)
 			if (opt.dhmlen < 0) {
 				goto usage;
 			}
+		} else if (strcmp(p, "aging") == 0) {
```

Comment:
This wll be shown as a problem at security test. How about modify this to strncmp?
```
} else if (strncmp(p, "aging", sizeof("aging") + 1) == 0) {
```

Reply:
Actually I know strncmp is more secure. but I consider the consistency  above logic. so I think strcmp is enough.

Reply:
I will fix